modules/test/playwright/tests/users-admin-web/customFieldLabelEscape.spec.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {loginTest} from '../../fixtures/loginTest';
+import {usersAndOrganizationsPagesTest} from '../../fixtures/usersAndOrganizationsPagesTest';
+
+export const test = mergeTests(loginTest(), usersAndOrganizationsPagesTest);
+
+test('LPD-29981 AUI Option Escape Label', async ({page, usersAndOrganizationsPage}) => {
+    await test.step('Add Custom Field to User', async () => {
+        await usersAndOrganizationsPage.goToUsers();
+
+        const testUser = page.getByRole('link', { name: 'Test Test' });
```

Comment:
Please see https://github.com/liferay-frontend/liferay-portal/pull/4313#discussion_r1687632458 in your other PR. The same logic applies here, almost all variables in this file are not named correctly.

Reply:
Will work on this, thank you!